docs/source/apps/cs2cs.rst

```diff
@@ -104,6 +104,10 @@ The following control parameters can appear in any order:
 
     List of all ellipsoids that can be selected with the *+ellps* parameters.
 
+.. option:: -lm
+
+    List of hard-coded prime meridians that can be selected with the *+pm* parameter.
```

Comment:
A note here stating that the list is hard-coded would be in order, I think. Basically the same as what is written for `PJ_PRIME_MERIDIANS`.

Reply:
Done.